CHANGES.md

```diff
@@ -190,6 +190,7 @@
   * `XMonad.Hooks.DynamicIcons`
 
     Added Dynamic Strings as `dynamicLogIconWithPP` based on a Workspaces Windows
+    Added ability to change to only show focused window
```

Comment:
When rendered, these are just concatenated without any punctuation; that looks weird.  Also I don't think this description is very discoverable (it still is the "new modules" section, so a sentence what the module does would be nice)